git.go

```diff
@@ -120,6 +123,25 @@ func (g *GitClient) Merge(sha string) error {
 	return nil
 }
 
+// GitCryptUnlock unlocks the repository using git-crypt
+func (g *GitClient) GitCryptUnlock(base64key string) error {
+	keyDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		return fmt.Errorf("failed to create temporary directory")
+	}
+	defer os.RemoveAll(keyDir)
+	decodedKey, err := base64.StdEncoding.DecodeString(base64key)
+	if err != nil {
+		return fmt.Errorf("failed to decode git-crypt key")
+	}
+	keyPath := filepath.Join(keyDir, "git-crypt-key")
+	ioutil.WriteFile(keyPath, decodedKey, 600)
```

Comment:
`git-crypt unlock` wants a path to a file, so the code above writes the key to a temporary directory (after decoding it). This is the same approach used by the official git resource.